lib/rules/require-aligned-multiline-params.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Enforces indentation of parameters in multiline functions
+ *
+ * Types: `Boolean`, `String`, `Number`
+ *
+ * Values:
+ *  - `true` to require parameters are aligned with the body of the function
+ *  - `'firstParam'` to require parameters to be aligned with the first parameter
+ *  - `Number` for the number of columns the parameters should be indented past the function body
+ *
+ * #### Example
+ *
+ * ```js
+ * "disallowSpaceBeforeComma": true
+ * ```
+ *
+ * ##### Valid for `true`
+ *
+ * ```js
+ * var test = function(one, two,
+ *   three, four, five,
+ *   six, seven, eight) {
+ *   console.log(a);
+ * };
+ * ```
+ *
+ * ##### Valid for `2`
+ *
+ * ```js
+ * var test = function(one, two,
+ *     three, four, five,
+ *     six, seven, eight) {
+ *   console.log(a);
+ * };
+ * ```
+ *
+ * ##### Valid for `'firstParam'`
+ *
+ * ```js
+ * var test = function(one, two,
+ *                     three, four, five,
+ *                     six, seven, eight) {
+ *   console.log(a);
+ * };
+ * ```
+ *
+ */
+
+var assert = require('assert');
+
+module.exports = function() {
+};
+
+module.exports.prototype = {
+
+    configure: function(option) {
+        if (typeof option === 'number') {
```

Comment:
In that case, I'd prefer to leave it as is. The tests/logic is already done, and I don't think this is the best example to use it in. Here, we're only validating the type/value of the `option` parameter. It would feel like overkill to use that library in this scenario, as we're not checking JSON.